06_PDDE_Checker.R

```diff
@@ -227,10 +234,93 @@ zero_utilization <- Project0 %>%
   mutate(Detail = "") %>%
   select(all_of(PDDEcols))
 
-##### For later-------
-# Incompatible Funding Source and Project Type Funding Source X can only be used with Project Type Y. Project Type A can only be used with Funding Source B. (this will need a lot more detail, hold on this one)
+# For later.. -------------------------------------------------------------
+
+# Incompatible Funding Source and Project Type Funding Source X can only be used
+# with Project Type Y. Project Type A can only be used with Funding Source B.
+# (this will need a lot more detail, hold on this one)
+
+
+# Overlapping participations ----------------------------------------------
+
+overlapping_ce_participation <- CEParticipation %>%
+  left_join(Project0 %>% select(ProjectID, OrganizationName, ProjectName),
+            by = "ProjectID") %>%
+  group_by(ProjectID) %>%
+  arrange(CEParticipationStatusStartDate) %>%
+  mutate(PreviousCEParticipationID = lag(CEParticipationID),
+         PreviousCEStart = lag(CEParticipationStatusStartDate),
+         PreviousCEEnd = lag(CEParticipationStatusEndDate)) %>%
+  ungroup() %>%
+  filter(!is.na(PreviousCEParticipationID)) %>%
+  mutate(ParticipationPeriod =
+           interval(
+             CEParticipationStatusStartDate,
+             coalesce(CEParticipationStatusEndDate, meta_HUDCSV_Export_Date)),
```

Comment:
I thought we decided to generally use Export_End, rather than Export_Date. Is there a reason we're using the latter here?